app/src/main/res/layout/fragment_events_manager_creator.xml

```diff
@@ -0,0 +1,346 @@
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why do you use `android:transitionGroup` for every widget. DO NOT DO THIS.

Reply:
On my cell phone, the animation was a bit strange, I thought that doing this would solve it, but it didn't change, and I didn't remove it from the code